app/src/androidTest/java/org/mozilla/fenix/ui/LoginsTest.kt

```diff
@@ -0,0 +1,336 @@
+package org.mozilla.fenix.ui
+
+import android.os.Build
+import android.view.autofill.AutofillManager
+import androidx.core.net.toUri
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.uiautomator.UiDevice
+import okhttp3.mockwebserver.MockWebServer
+import org.junit.After
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.mozilla.fenix.customannotations.SmokeTest
+import org.mozilla.fenix.ext.settings
+import org.mozilla.fenix.helpers.AndroidAssetDispatcher
+import org.mozilla.fenix.helpers.HomeActivityIntentTestRule
+import org.mozilla.fenix.helpers.TestAssetHelper
+import org.mozilla.fenix.helpers.TestHelper
+import org.mozilla.fenix.helpers.TestHelper.exitMenu
+import org.mozilla.fenix.ui.robots.browserScreen
+import org.mozilla.fenix.ui.robots.navigationToolbar
+
+class LoginsTest {
+    private lateinit var mDevice: UiDevice
+    private lateinit var mockWebServer: MockWebServer
+
+    @get:Rule
+    val activityTestRule = HomeActivityIntentTestRule.withDefaultSettingsOverrides(skipOnboarding = true)
+
+    @Before
+    fun setUp() {
+        mDevice = UiDevice.getInstance(InstrumentationRegistry.getInstrumentation())
+        TestHelper.appContext.settings().userOptOutOfReEngageCookieBannerDialog = true
+        mockWebServer = MockWebServer().apply {
+            dispatcher = AndroidAssetDispatcher()
+            start()
+        }
+
+        if (Build.VERSION.SDK_INT == Build.VERSION_CODES.R) {
```

Comment:
Shouldn't this be `>=`, as this will currently skip over on Android 11+

https://developer.android.com/reference/android/os/Build.VERSION_CODES#R

But, looking at  https://developer.android.com/reference/android/view/autofill/AutofillManager was added in `API 26`, I'm wondering if we should just set this for `>=` `Build.VERSION_CODES.O` 

https://developer.android.com/reference/android/os/Build.VERSION_CODES#O